lib/ROCrates.Net.Tests/TestEntity.cs

```diff
@@ -5,15 +5,18 @@
 
 namespace ROCrates.Tests;
 
-public class TestRoCrateEntity
+public class TestEntity
 {
+  private ROCrate _roCrate = new();
+  private string _jsonLd =
+    "{\"@id\": \"./\",\"identifier\": \"https://doi.org/10.4225/59/59672c09f4a4b\",\"@type\": \"Dataset\", \"randomNumber\": 123, \"datePublished\": \"2017\",\"name\": \"Data files associated with the manuscript:Effects of facilitated family case conferencing for ...\",\"description\": \"Palliative care planning for nursing home residents with advanced dementia ...\",\"license\": {\"@id\": \"https://creativecommons.org/licenses/by-nc-sa/3.0/au/\"}}";
```

Comment:
I would use verbatim or raw string literals here to preserve the whitespace and simplify escaping to make it at all readable.

verbatim string literals (`@""`) allows you to not need escapes (or to escape double quotes with double double quotes `""`

In .NET 7+ (C#11) you can use [raw string literals](https://thecodeblogger.com/2022/09/17/c-11-what-are-raw-string-literals/) which don't even need to escape double quotes.
```suggestion
    @"
		{
			""@id"": ""./"",
			""identifier"": ""https://doi.org/10.4225/59/59672c09f4a4b"",
			""@type"": ""Dataset"",
			""randomNumber"": 123,
			""datePublished"": ""2017"",
			""name"": ""Data files associated with the manuscript:Effects of facilitated family case conferencing for ..."",
			""description"": ""Palliative care planning for nursing home residents with advanced dementia ..."",
			""license"": {
				""@id"": ""https://creativecommons.org/licenses/by-nc-sa/3.0/au/""
			}
		}";
```